ContentPatcher/docs/i18n/zh/author-guide/tokens.md

```diff
@@ -0,0 +1,2011 @@
+}
+```
+
+</dd>
+
+<dt>固定键到底是什么？</dt>
+<dd>
+
+无固定键时，每个令牌独立选择：
+```txt
+{{Random: hood, jacket, raincoat}} = raincoat
+{{Random: hood, jacket, raincoat}} = hood
+{{Random: hood, jacket, raincoat}} = jacket
+```
+
+相同固定键时，保持同步：
+```txt
+{{Random: hood, jacket, raincoat |key=outfit}} = hood
+{{Random: hood, jacket, raincoat |key=outfit}} = hood
+{{Random: hood, jacket, raincoat |key=outfit}} = hood
+```
+
+固定键同步内部随机数，相同选项+相同键=相同值。
+
+```txt
+{{Random: hood, jacket, raincoat |key=outfit}} = 217437 modulo 3 choices = index 0 = hood
+{{Random: hood, jacket, raincoat |key=outfit}} = 217437 modulo 3 choices = index 0 = hood
+{{Random: hood, jacket, raincoat |key=outfit}} = 217437 modulo 3 choices = index 0 = hood
+```
+
+若选项顺序不同，相同索引对应不同值：
+```txt
+{{Random: hood, jacket, raincoat |key=outfit}} = 217437 modulo 3 choices = index 0 = hood
+{{Random: jacket, raincoat, hood |key=outfit}} = 217437 modulo 3 choices = index 0 = jacket
+```
+</dd>
+</dl>
+<a name="advanced"></a>
+
+## 进阶 
+<a name="query-expressions"></a>
+
+## 查询表达式 
+_查询表达式_ 是一组可计算为数字，`true`/`false`或文本的算术和逻辑表达式。
+
+### 用法
+使用`Query`执行查询表达式。可用作占位符或条件，可包含嵌套令牌：
+```js
+{
+   "Format": "2.7.0",
+   "Changes": [
+      {
+         "Action": "EditData",
+         "Target": "Characters/Dialogue/Abigail",
+         "Entries": {
+            "Mon": "Hard to imagine you only arrived {{query: {{DaysPlayed}} / 28}} months ago!"
+         },
+         "When": {
+            "query: {{Hearts:Abigail}} >= 10": true
+         }
+      }
+   ]
+}
+```
+
+文本值需用单引号包裹（包括返回文本的令牌）：
+```js
+"Query: '{{Season}}' = 'spring'": true
+```
+
+表达式不区分大小写，包括文本比较。
+
+### 注意事项
+查询表达式功能强大，但需注意：
+
+* 查询表达式 **很难验证** 无效表达式通常不会预先警告，仅在应用补丁时失败。需仔细测试表达式的功能，可以用[`patch parse`](troubleshooting.md#parse)检查新表达式。
+* 查询表达式 **可能错误读取文本** 例如，玩家名含单引号`D'Artagnan`，时，以下表达式会因语法错误失败：
+  ```js
+  "Query: '{{PlayerName}}' LIKE 'D*'": true // 'D'Artagnan' LIKE 'D*'
+  ```
+* 可能返回技术性错误信息。
+* 可能降低内容包可读性。
+
+尽可能使用非表达式功能。例如：
+
+<table>
+<tr>
+<th>使用查询表达式</th>
+<th>不使用查询表达式</th>
+</tr>
+<tr>
+<td>
+
+```js
+"When": {
+   "Query: {{Time}} >= 0600 AND {{Time}} <= 1200": true
+}
+```
+
+</td>
+<td>
+
+```js
+"When": {
+   "Time": "{{Range: 0600, 1200}}"
+}
+```
+
+</td>
+</tr>
+</table>
+
+### 运算符
+支持的运算符如下：
+
+* 算术运算（如`5 + 5`）：
+
+  运算符 | 效果
+  -------- | ---------
+  \+       | 加
+  \-       | 减
+  \*       | 乘
+  /        | 除
+  %        | 取模
+  ()       | 组合
+
+* 比较值（如`5 < 10`）：
+
+  运算符 | 效果
+  -------- | ---------
+  `<`      | 小于
+  `<=`     | 小于等于
+  `>`      | 大于
+  `>=`     | 大于等于
+  `=`      | 等于
+  `<>`     | 不等于
+
+* 逻辑运算符：
+
+  运算符 | 效果
+  -------- | ------
+  `AND`    | 与，两者都为真，如`{{Time}} >= 0600 AND {{Time}} <= 1200`。
+  `OR`     | 或，至少一者为真，如`{{Time}} <= 1200 OR {{Time}} >= 2400`。
+  `NOT`    | 非，取反，如`NOT {{Time}} > 1200`。
+
+* 使用`()`组合运算避免歧义：
+
+  ```js
+  "Query: ({{Time}} >= 0600 AND {{Time}} <= 1200) OR {{Time}} > 2400": true
+  ```
+
+* 检查值是否`IN`或`NOT IN`列表中：
```

Comment:
> 检查值是否`IN`或`NOT IN`列表中：

The original text used `IN` and `NOT IN` grammatically as part of the sentence, which probably doesn't make sense in Chinese. I suggest translating it as something like "_Check whether a value is in (`IN`) or not in (`NOT IN`) a list_".